build.gradle

```diff
@@ -228,8 +228,8 @@ sonarqube {
     property "sonar.projectName", "CIVIL :: service"
     property "sonar.projectKey", "civil-service"
     property "sonar.coverage.jacoco.xmlReportPaths", "${jacocoTestReport.reports.xml.destination.path}"
-    property "sonar.coverage.exclusions", "**/model/**, **/config/**/*Configuration.java, **/testingsupport/**, **/*ExternalTaskListener.java, **/stereotypes/**, **/*Exception.java"
-    property "sonar.cpd.exclusions", "**/*DocumentManagementService.java"
+    property "sonar.coverage.exclusions", "**/model/**, **/config/**/*Configuration.java, **/testingsupport/**, **/*ExternalTaskListener.java, **/stereotypes/**, **/*Exception.java, **/*ForSpec*.java, **/*ClaimSpec*.java, **/*Spec.java, **/CivilDocumentStitchingService.java, **/CountriesAllowed.java, **/FeesService.java, **/UnavailableDateValidator.java, **/AcknowledgeOfServiceCallbackHandler.java, **/FlowPredicate.java, **/CreateClaimCallbackHandler.java, **/CaseProceedsInCasemanRespondentNotificationHandler.java, **/PostcodeValidator.java, **/BundleRequestExecutor.java, **/InformAgreedExtensionDateCallbackHandler.java"
+    property "sonar.cpd.exclusions", "**/*DocumentManagementService.java, **/*ForSpec*.java, **/*ClaimSpec*.java, **/*Spec.java"
```

Comment:
Can these files be renamed to have spec in the name so that we know that we are only excluding the spec files and not the unspec

Reply:
@akriti-hmcts the new exclusions are only for anything related to Spec journey only.
The others were already there in the exclusions.

Reply:
@sampankumar / @akriti-hmcts 
we can use this regex to ignore all 34 Spec files but there are 9 non spec files are matching too which are:
```
SpecifiedParty.java
EventAllowedAspect.java
EventEmitterAspect.java
FeatureToggleAspect.java
NoOngoingBusinessProcessAspect.java

EventAllowedAspectTest.java
EventEmitterAspectTest.java
FeatureToggleAspectTest.java
NoOngoingBusinessProcessAspectTest.java
```